ccl/streamingccl/streamclient: TestPartitionedStreamReplicationClient failed

ccl/streamingccl/streamclient.TestPartitionedStreamReplicationClient [failed](https://tanzanite.cluster.engflow.com/invocations/default/70a9b994-2ea2-45c6-9514-10852336ae9f?testReportRun=24&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3N0cmVhbWluZ2NjbC9zdHJlYW1jbGllbnQ6c3RyZWFtY2xpZW50X3Rlc3Q=) on master @ [dab086d9d1d3355ad381ec493c73cabfa8d6b046](https://github.com/cockroachdb/cockroach/commits/dab086d9d1d3355ad381ec493c73cabfa8d6b046):


```
=== RUN   TestPartitionedStreamReplicationClient
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestPartitionedStreamReplicationClient1725410578
    test_log_scope.go:81: use -show-logs to present logs inline
    replication_helpers.go:238: Replication helper seed -7880645388834023797
    replication_helpers.go:298: using inline certificates
    sql_runner.go:121: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:134
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:369
        	            				github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient_test/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go:353
        	Error:      	Received unexpected error:
        	            	query 'SELECT status FROM [SHOW JOBS] WHERE job_id = 933225262258421761': expected:
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:372
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:134
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:369
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient_test.TestPartitionedStreamReplicationClient
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient_test/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go:353
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT status FROM [SHOW JOBS] WHERE job_id = 933225262258421761': expected:
        	            	  | succeeded
        	            	  |
        	            	  | got:
        	            	  | failed
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: outputs.zip/logTestPartitionedStreamReplicationClient1725410578
--- FAIL: TestPartitionedStreamReplicationClient (53.88s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=24</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestPartitionedStreamReplicationClient.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-35267

ccl/streamingccl/streamclient.TestPartitionedStreamReplicationClient [failed](https://tanzanite.cluster.engflow.com/invocations/default/2d226013-7a59-481e-aa0e-289a94a4e1bb?testReportRun=5&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY2NsL3N0cmVhbWluZ2NjbC9zdHJlYW1jbGllbnQ6c3RyZWFtY2xpZW50X3Rlc3Q=) on master @ [f2744bbbcd88ccfb09a6a0af0a8b2342302627ef](https://github.com/cockroachdb/cockroach/commits/f2744bbbcd88ccfb09a6a0af0a8b2342302627ef):


```
=== RUN   TestPartitionedStreamReplicationClient
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestPartitionedStreamReplicationClient148533322
    test_log_scope.go:81: use -show-logs to present logs inline
    replication_helpers.go:238: Replication helper seed 114176349782928418
    sql_runner.go:121: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:134
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:369
        	            				github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient_test/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go:353
        	Error:      	Received unexpected error:
        	            	query 'SELECT status FROM [SHOW JOBS] WHERE job_id = 933510503972864001': expected:
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:372
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:134
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:369
        	            	  | github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient_test.TestPartitionedStreamReplicationClient
        	            	  | 	github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient_test/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go:353
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT status FROM [SHOW JOBS] WHERE job_id = 933510503972864001': expected:
        	            	  | succeeded
        	            	  |
        	            	  | got:
        	            	  | failed
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: outputs.zip/logTestPartitionedStreamReplicationClient148533322
--- FAIL: TestPartitionedStreamReplicationClient (52.78s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=5</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestPartitionedStreamReplicationClient.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go
+++ b/pkg/ccl/streamingccl/streamclient/partitioned_stream_client_test.go
@@ -345,7 +345,7 @@ INSERT INTO d.t2 VALUES (2);
 	// Makes producer job exit quickly.
 	h.SysSQL.ExecMultiple(t, `
 SET CLUSTER SETTING stream_replication.stream_liveness_track_frequency = '200ms'`,
-		`SET CLUSTER SETTING stream_replication.job_liveness_timeout = '100ms'`)
+		`SET CLUSTER SETTING stream_replication.job_liveness_timeout = '1s'`)
 	rps, err = client.Create(ctx, testTenantName, streampb.ReplicationProducerRequest{})
 	require.NoError(t, err)
 	streamID = rps.StreamID

